js/feed.js

```diff
@@ -1,141 +1,213 @@
+            field.payload.commits.map(function (commit) {
+              content += commit.message + "<br>";
+            })
+            link = field.payload.compare;
+            break;
+        }
+
+        if (!eventType)
+          return; // don't display if event not handled above
+
+        $(".feed-container").append(
+          $("<div/>", { "class": "event" }).append(
+            $("<div/>", { "class": "label" }).append(
+              $("<a/>", { "href": "https://github.com/" + field.actor.display_login, "target": "_blank" }).append(
+                $("<img>", { "src": field.actor.avatar_url })
               )
-            ).append(
-              $("<div/>", { "class": "ui small message" }).append(
-                $("<div/>", { "class": "header" }).append(
-                  $("<a/>", { "href": link, "style": "text-decoration: none;" }).append(
-                    $("<h3/>").text(header)
-                  )
+            )
+          ).append(
+            $("<div/>", { "class": "content" }).append(
+              $("<div/>", { "class": "summary" }).append(
+                $("<a/>", { "href": "https://github.com/" + field.actor.display_login, "target": "_blank" }).text(field.actor.display_login)
+              ).append(" " + eventType + " ").append(
+                $("<a/>", { "href": "https://github.com/" + field.repo.name, "target": "_blank" }).text(field.repo.name)
+              ).append(
+                $("<div/>", { "class": "date", "style": "font-size: 100%;" }).append(
+                  " " + timeSince(new Date(field.created_at))
                 )
               ).append(
-                content ? replaceURLsWithLinks(new showdown.Converter().makeHtml(content)) : ""
+                $("<div/>", { "class": "ui small message" }).append(
+                  $("<div/>", { "class": "header" }).append(
+                    $("<a/>", { "href": link, "style": "text-decoration: none;", "target": "_blank" }).append(
+                      $("<span/>", { "class": "iconify", "data-icon": icon, "style": "color: " + iconColor })
+                    ).append(
+                      $("<span/>").text(" " + header)
+                    )
+                  )
+                ).append(
+                  content ? converter.makeHtml(content) : ""
+                )
               )
-            )
-          ))
-      ).append(
-        $("<div/>", { "class": "ui divider" })
-      );
+            ))
+        ).append(
+          $("<div/>", { "class": "ui divider" })
+        );
+      });
+
+      if (loadMoreButton) {
+        $(".feed-container").append(
+          $("<button/>", { "class": "fluid ui button", "id": "loadMore" }).text("Load more")
+        )
+      }
 
       $(window).trigger("resize");
-    });
-  })
-}());
+
+      $("#loadMore").click(function () {
+        pageToLoad++;
+        createActivityFeed(30, true);
+        $("#loadMore").remove();
```

Comment:
why do you remove it? Is it because the function creates it again?
Can you have the function create the button only if needed? So you can avoid removing it here, which would make the code easier to maintain

Reply:
I remove it because the button will stay and the new content will be appended after it. What would you recommend to do here?
![image](https://user-images.githubusercontent.com/34558138/49443047-5309ac80-f799-11e8-84e7-daadea93f136.png)